util/db/repository.go

```diff
@@ -58,22 +58,12 @@ func (db *db) CreateRepository(ctx context.Context, r *appsv1.Repository) (*apps
 		return nil, status.Errorf(codes.AlreadyExists, "repository '%s' already exists", r.Repo)
 	}
 
-	data := make(map[string][]byte)
```

Comment:
how come deleted?

Reply:
Apparently, `data` is just not used anywhere. Did I miss something?